.github/workflows/release.yml

```diff
@@ -110,13 +110,13 @@ jobs:
         run: ansible-galaxy collection publish --api-key=${{ secrets.GALAXY_INFRA_KEY }} ${{ steps.build.outputs.tar_file }}
 
       - name: Update dependency for Automation Hub
-        run: "sed -i 's/fedora\.linux_system_roles/redhat.linux_system_roles/g' galaxy.yml"
```

Comment:
This might be one of those times where using the character class `[.]` might be easier than using the escape - so 

```
        run: "sed -i 's/fedora[.]linux_system_roles/redhat.linux_system_roles/g' galaxy.yml"
```